### PR TITLE
ONBOARD-5 Basic project documentation

### PR DESCRIPTION
**Pull request by taraspogrebniak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+Follow the instructions described [here](http://www.scala-sbt.org/0.13/docs/Installing-sbt-on-Linux.html)
+
+### Open JDK 1.8 or Oracle JDK 1.8
+For example, Oracle JDK can be installed like:
+```
+sudo add-apt-repository ppa:webupd8team/java
+sudo apt-get update
+sudo apt-get install oracle-java8-installer
+sudo apt-get install oracle-java8-set-default
+```
```
Comment (ngraca) [inline README.md]: MacOS:
Download from
http://www.oracle.com/technetwork/java/javase/downloads/index.html

File: README.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+- taras.pogrebniak@springernature.com
+
+## Source
+In order to start working woth `nemo-on-board` application just clone the current repository by executing the command `git clone git@github.com:springernature/nemo-on-board.git`
+
+## System dependencies
+### SBT 
+Follow the instructions described [here](http://www.scala-sbt.org/0.13/docs/Installing-sbt-on-Linux.html)
+
+### Open JDK 1.8 or Oracle JDK 1.8
```
Comment (ngraca) [inline README.md]: This should be at the beginning, before the SBT installation

File: README.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+
+##Contact
+- nuno.graca@springernature.com
+- federico.quintas@springernature.com
+- taras.pogrebniak@springernature.com
+
+## Source
+In order to start working woth `nemo-on-board` application just clone the current repository by executing the command `git clone git@github.com:springernature/nemo-on-board.git`
+
+## System dependencies
```
Comment (ngraca) [inline README.md]: `### Brew
$ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
$ brew update'

File: README.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
+### SBT 
+Follow the instructions described [here](http://www.scala-sbt.org/0.13/docs/Installing-sbt-on-Linux.html)
+
```
Comment (ngraca) [inline README.md]: MacOS:
$ brew install sbt

File: README.md [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+For example, Oracle JDK can be installed like:
+```
+sudo add-apt-repository ppa:webupd8team/java
+sudo apt-get update
+sudo apt-get install oracle-java8-installer
+sudo apt-get install oracle-java8-set-default
+```
+
+### Scala
+Checkout the recent release of scala [here](https://www.scala-lang.org/download/)
```
Comment (ngraca) [inline README.md]: MacOs:

$ brew install scala

File: README.md [hunk 6]
```diff
@@ -1,3 +1,43 @@
...
+## Description
+This project is aimed to make new developers familiar with the infrastructure used by Nemo project, deployment scenarios, application requirements etc.
+
+##Contact
+- nuno.graca@springernature.com
+- federico.quintas@springernature.com
+- taras.pogrebniak@springernature.com
+
+## Source
+In order to start working woth `nemo-on-board` application just clone the current repository by executing the command `git clone git@github.com:springernature/nemo-on-board.git`
```
Comment (FedericoQuintas) [inline README.md]: *with

**Pull request merged**